trigger/task.go

```diff
@@ -260,6 +260,9 @@ func (t *taskTriggers) makeData(sub *event.Subscription, pastTenseOverride strin
 		displayName = t.task.DisplayTask.DisplayName
 		status = t.task.DisplayTask.Status
 	}
+	if testNames != "" {
+		displayName += fmt.Sprintf("(%s)", testNames)
```

Comment:
I think this should be `" (%s)"` so that there's a space between the display name and the first parens.

Reply:
changed